packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1610,6 +1610,7 @@ function activatePurposeVersionFromOverQuotaSuspendedLogic(
 
   const updatedPurpose: Purpose = {
     ...purpose.data,
+    suspendedByConsumer: false,
```

Comment:
Is this endpoint called by the producer? If so, are we sure this value has to be changed? Maybe another endpoint (called by the consumer) should update that flag.

cc @Carminepo2

Reply:
If I'm not missing something, this function is called by the consumer when tries to activate a suspended overquota purpose.
Btw I'm not entirely sure the fix should be here because in this point the current version is still `SUSPENDED`, but we lose the flag.

Considering that when a `WAITING_FOR_APPROVAL` version gets approved it always pass to `ACTIVE`, I think it's better to remove the flag during that operation.